_includes/manuals/nightly/4.1.3_trends.md

```diff
@@ -1,3 +1,5 @@
 
 Trends and statistics are moved to the separate plugin.
 See [manual](/plugins/foreman_statistics) for the reference.
+
+There is a rake task `foreman-rake purge:trends` for users who are not planning to use trends and statistics anymore and wish to clean up database.
```

Comment:
This should also be added in 2.2 manual (we shipped the task in rc3), perhaps also mention it in the 2.2 upgrade warning section mentioning the extraction.

Reply:
Done